package-lock.json

```diff
@@ -12,6 +12,7 @@
         "@uniswap/default-token-list": "^2.0.0",
         "@uniswap/permit2-sdk": "^1.2.0",
         "@uniswap/router-sdk": "^1.3.0",
+        "@uniswap/sdk-core": "^3.2.6",
```

Comment:
The amount of changes in `package-lock.json` is way too high. I expect much smaller number of changes. With this many dependencies changes, we might have unexpected side effects when getting the newer sor adopted in [interface](https://github.com/Uniswap/interface) and [routing-api](https://github.com/Uniswap/routing-api)

Reply:
good idea, I fixed it by adding an overrides part to package.json, to force the transitive dependency of sdk-core to be version 3.2.6.